src/vs/workbench/parts/extensions/browser/extensionsActions.ts

```diff
@@ -358,6 +358,7 @@ export class ManageExtensionAction extends Action {
 				instantiationService.createInstance(EnableGloballyAction, localize('enableAlwaysAction.label', "Enable (Always)"))
 			],
 			[
+				instantiationService.createInstance(AddToWorkspaceRecommendationsAction, localize('addToWorkspaceRecommendationsAction.label', "Add to Workspace Recommendations")),
```

Comment:
How about having a short label - `Recommend (Workspace)`  and `Unrecommend (Workspace)`. Not sure `Workspace` word is necessary since a user can recommend only to Workspace, but having it makes it complete.